File: CourseCodesAPI/CourseCodesAPI/Models/CourseCreateRequest.cs
using System;

namespace CourseCodesAPI.Models
{
	public class CourseCreateRequest
	{
		public Guid InstructorId { get; set; }
		public string CourseName { get; set; }
		public string Term { get; set; }
		public string Section { get; set; }
		public int Capacity { get; set; }
	}
}

File: coursecodesapp/src/api/models/CourseCreateRequest.js
export class CourseCreateRequest {
	constructor({ instructorId, courseName, term, section, capacity }) {
		this.instructorId = instructorId;
		this.courseName = courseName;
		this.term = term.toUpperCase();
		this.section = section.toUpperCase();
		this.capacity = parseInt(capacity);
	}
}
